File: CS/DisableFloatingPanelTransparencyBehavior/DisableFloatingPanelTransparencyBehavior.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Windows.Interactivity;
using DevExpress.Xpf.Docking;
using System.Windows;

namespace DisableFloatingPanelTransparencyBehavior
{
	public class DisableFloatingPanelTransparencyBehavior: Behavior<DockLayoutManager>
	{
		private ResourceDictionary floatPaneStyles;
		protected override void OnAttached()
		{
			base.OnAttached();
			AssociatedObject.SetValue(DockLayoutManager.AllowFloatGroupTransparencyProperty, false);
			floatPaneStyles = new ResourceDictionary();
			floatPaneStyles.Source = new Uri(@"pack://application:,,,/DisableFloatingPanelTransparencyBehavior;component/themes/floatpane.xaml");			
			AssociatedObject.Resources.MergedDictionaries.Add(floatPaneStyles);
		}

		protected override void OnDetaching()
		{
			AssociatedObject.Resources.MergedDictionaries.Remove(floatPaneStyles);
			base.OnDetaching();
		}
	}
}

File: VB/DisableFloatingPanelTransparencyBehavior/DisableFloatingPanelTransparencyBehavior.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports System.Windows.Interactivity
Imports DevExpress.Xpf.Docking
Imports System.Windows

Namespace DisableFloatingPanelTransparencyBehavior
	Public Class DisableFloatingPanelTransparencyBehavior
		Inherits Behavior(Of DockLayoutManager)
		Private floatPaneStyles As ResourceDictionary
		Protected Overrides Sub OnAttached()
			MyBase.OnAttached()
			AssociatedObject.SetValue(DockLayoutManager.AllowFloatGroupTransparencyProperty, False)
			floatPaneStyles = New ResourceDictionary()
			floatPaneStyles.Source = New Uri("pack://application:,,,/DisableFloatingPanelTransparencyBehavior;component/themes/floatpane.xaml")
			AssociatedObject.Resources.MergedDictionaries.Add(floatPaneStyles)
		End Sub

		Protected Overrides Sub OnDetaching()
			AssociatedObject.Resources.MergedDictionaries.Remove(floatPaneStyles)
			MyBase.OnDetaching()
		End Sub
	End Class
End Namespace
